tools/run_tests/helper_scripts/build_python.sh

```diff
@@ -230,10 +228,11 @@ pip_install_dir "$ROOT/src/python/grpcio_testing"
 
 # Build/install tests
 # shellcheck disable=SC2261
-pip_install coverage==7.2.0 oauth2client==4.1.0 \
+pip_install coverage>=7.9.0 oauth2client==4.1.0 \
```

Comment:
to avoid `>` being interpreted as a redirect

```suggestion
pip_install "coverage>=7.9.0" oauth2client==4.1.0 \
```